models/experimental/gemma3/tt/text_model.py

```diff
@@ -452,6 +588,15 @@ def forward(
             elif activation_dtype is not None and x.dtype != activation_dtype:
                 x = ttnn.typecast(x, activation_dtype)
```

Comment:
Nitpick, naming the masks first makes it a bit easier to read, for instance:
```suggestion
            causal_mask = None
            if attention_masks is not None:
                sliding_causal_mask, global_causal_mask = attention_masks
                causal_mask = sliding_causal_mask if getattr(layer.attention, "is_sliding", False) else global_causal_mask
```